minai_plugin/customintegrations.php

```diff
@@ -96,6 +98,11 @@ function ProcessIntegrations() {
         );
         $MUST_DIE=true;
     }
+    if (isset($GLOBALS["gameRequest"]) && $GLOBALS["gameRequest"][0] == "updateThreadsDB") {
+        file_put_contents("my_logs.txt", "\n\ncustom integrations: updateThreadsDB!!!\n\n", FILE_APPEND);
```

Comment:
We should probably write a little logger implementation. I've been meaning to do this anyways, and switch all of the mod's logging to our own file.

Reply:
yeah it's temporary, I'll clean it when it's ready to merge :)